contributors.md

```diff
@@ -0,0 +1,61 @@
+# TSC
+
+Ali Javadi
+Bettina Heim
+Blake Johnson
+Lev Bishop
+Philipp Schindler
+Steven Heidel
+
+
+# Contributors by commit
+
+Aaron Berdy
+Abeer Vaishnav
+Andrew Cross
+Bruno Schmitt
+Carter Snook
+Colm Ryan
+Daniel Bochen Tan
+Hiroshi Horii
+Ismael Faro
+Jack Woehr
+Jake Lishman
+Jay Gambetta
+Jean-Christophe Jaskula
+Jeff Heckey
+Li Chen
+Luciano Bello
+Matthew Treinish
+Michael Healy
+Noah Wood
+Pranav Gokhale
+Prasahnt Sivarajah
+Salvadore de la Puente
+Thomas Alexander
+Travis Scholten
+Yunong Shikj
```

Comment:
@jakelishman I believe Yunong's last name is "Shi".

Reply:
oh whoops thanks - that's what I originally typed, then fidgeted around in vim without exiting insert mode!